packages/web3-providers/src/types/Trending.ts

```diff
@@ -183,7 +183,7 @@ export namespace TrendingAPI {
         /** Get all related coins with a keyword. */
         getCoinsByKeyword: (chainId: ChainId, keyword: string) => Promise<Coin[]>
         /** Get coin info by address. */
-        getCoinInfoByAddress: (address: string) => Promise<CoinInfo | undefined>
+        getCoinInfoByAddress: (address: string, chainId?: number) => Promise<CoinInfo | undefined>
```

Comment:
Refactor it like other methods.

Reply:
`address` is the major parameter, and `chainId` is optional.

Reply:
After check the implementation, I’d use customChainIds as the name of the second parameter. And the type of it will be ChainId[]. What do you think?

Reply:
customChainIds would be `number`.

> And the type of it will be ChainId[]. 

For now it only needs to check one chain, if we need to check more than one in the future, we can convert it to array accordingly.